src/openassetio-python/package/openassetio/hostApi/Manager.py

```diff
@@ -430,6 +436,23 @@ def getWithRelationship(
 
         @param context Context The calling context.
 
+        @param successCallback Callback that will be called for each
+        successful relationship query. It will be given the
+        corresponding index of the entity reference in
+        `entityReferences` along with a list of entity references for
+        entities that have the specified relationship. If there are no
+        relations, an empty list will be supplied. The callback will be
```

Comment:
[Consider][Minor]

Maybe just a quirk I have, but I continually find the mapping between the concept of a `Relationship` and the argument `relationshipTraitsData` awkward, as though the argument is saying "These are the traits datas that belong to, that define the relationship, but they are not the relationship", in the same way you might talk about "BookPages" and "Book" as different but essentially the same thing.

Long story short, maybe this ?

```suggestion
        `entityReferences` along with a list of entity references for
        entities that have the relationship specified by \p relationshipTraitsData
        If there are no relations, an empty list will be supplied. The callback will be
```

Reply:
Applied as diff, but then reworded as part of [this](https://github.com/OpenAssetIO/OpenAssetIO/pull/935/commits/d575d22518fb6a9167edd5f72ba3c57a77b8e4ea).

Reply:
:+1: